common/sgx/verifier.c

```diff
@@ -32,12 +33,12 @@ typedef oe_mutex oe_mutex_t;
 #define OE_MUTEX_INITIALIZER OE_H_MUTEX_INITIALIZER
 #endif
 
-static const oe_uuid_t _local_uuid = {OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION};
-static const oe_uuid_t _ecdsa_uuid = {OE_FORMAT_UUID_SGX_ECDSA_P256};
-static const oe_uuid_t _ecdsa_report_uuid = {
-    OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT};
-static const oe_uuid_t _ecdsa_quote_uuid = {
-    OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE};
+static const oe_uuid_t _uuid_sgx_local = {OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION};
```

Comment:
```suggestion
static const oe_uuid_t _uuid_sgx_local_attestation = {OE_FORMAT_UUID_SGX_LOCAL_ATTESTATION};
```
Make variable match define, like the others do.

Reply:
Thanks! I was hoping for shorter local variable names. Will update.